espnet2/bin/diar_inference.py

```diff
@@ -159,6 +160,36 @@ def __call__(
 
         return spk_prediction
 
+    @staticmethod
+    def from_pretrained(
+        model_tag: Optional[str] = None,
+        **kwargs: Optional[Any],
+    ):
+        """Build DiarizeSpeech instance from the pretrained model.
+
+        Args:
+            model_tag (Optional[str]): Model tag of the pretrained models.
+                Currently, the tags of espnet_model_zoo are supported.
+
+        Returns:
+            DiarizeSpeech: DiarizeSpeech instance.
+
+        """
+        if model_tag is not None:
+            try:
+                from espnet_model_zoo.downloader import ModelDownloader
+
+            except ImportError:
+                logging.error(
+                    "`espnet_model_zoo` is not installed. "
+                    "Please install via `pip install -U espnet_model_zoo`."
+                )
+                raise
+            d = ModelDownloader()
+            kwargs.update(**d.download_and_unpack(model_tag))
```

Comment:
The name of returned arguments are different.
https://github.com/espnet/espnet/blob/fb75313bd1ddf027e075ea7197448c929b176f5d/espnet2/bin/pack.py#L29-L32
Please change the names or add workaround solution.
https://github.com/kan-bayashi/espnet/blob/7f42a925b82dc722df7d93f08e07a1a62df390b3/espnet2/bin/enh_inference.py#L283-L285

Reply:
Thanks for the review. I corrected it as in `enh_inference.py`.

Reply:
Thanks!